helix-rest/src/main/java/org/apache/helix/rest/server/authValidator/DefaultAuthValidator.java

```diff
@@ -0,0 +1,10 @@
+package org.apache.helix.rest.server.authValidator;
+
+import javax.ws.rs.container.ContainerRequestContext;
+
+
+public class DefaultAuthValidator implements AuthValidator {
+  public boolean validate(ContainerRequestContext request) {
+    return true;
+  }
+}
```

Comment:
Do we have a plan to add "real" validators? 
I might be missing context, but It feels a little weird to have a standalone default class, which turns out to be a noop. A default method impl may be better depends on your use case.

Reply:
This is the default validator for open-source users that don't want to implement validators - the default behavior is no security.

Reply:
I see. In this case, the term "default" could be misleading, let's rename `DefaultAuthValidator` to e.g. `NoopAuthValidator`.
Alternatively, we can even remove this class entirely, and use default method with anonymous class like `new AuthValidator() { }`